Gordon360/Services/ScheduleService.cs

```diff
@@ -102,14 +102,18 @@ public async Task<IEnumerable<ScheduleViewModel>> GetScheduleFacultyAsync(string
         public async Task<IEnumerable<SessionCoursesViewModel>> GetAllCourses(string username)
         {
             var account = _context.ACCOUNT.FirstOrDefault(x => x.AD_Username == username);
-            var allSessions = _sessionService.GetAll();
-            var result = Enumerable.Empty<SessionCoursesViewModel>();
 
             if (account == null)
             {
                 throw new ResourceNotFoundException() { ExceptionMessage = "The account was not found." };
             }
-
+            var allSessions = _sessionService.GetAll();
+            var result = Enumerable.Empty<SessionCoursesViewModel>();
+            var allSchedule = _context.UserCourses
+                                  .Where(s => s.UserID == account.gordon_id)                  
+                                  .Select(s => (UserCoursesViewModel)s)
+                                  .AsEnumerable()        
+                                  .OrderByDescending(course => course.SessionCode);                        
```

Comment:
@mla04762 I know this is approved and merged, but it would be a great follow-on to change the order to put the latest (current or future) term first. (See issue #990).  Might be as simple as ".OrderByAscending(".

Reply:
I'll be unavailable for this week, but I'll try to fix this and push it sometime hopefully before the term starts